src/index.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>My leader Board</title>
 </head>
 <body>
   <header>
```

Comment:
[Optional] Currently, when the page loads, the scores do not load until the refresh button is pressed. Also, when a score it entered, it does not load on the page until the refresh button is pressed. I suggest that you look into making the load automatically.